pkgs/os-specific/linux/swapspace/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, utillinux }:
+
+stdenv.mkDerivation rec {
+  pname = "swapspace";
+  version = "1.17";
+
+  src = fetchFromGitHub {
+    owner = "Tookmund";
+    repo = "Swapspace";
+    rev = "v${version}";
+    sha256 = "06xvmyy1fp94h00k9nn929j00ca3w12fiz07wf9az6srxa8i4ndz";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  patchPhase = ''
+    sed -e 's@"mkswap"@"${utillinux}/bin/mkswap"@' \
+      -e 's@"/sbin/swapon"@"${utillinux}/bin/swapon"@' \
+      -e 's@"/sbin/swapoff"@"${utillinux}/bin/swapoff"@' \
+      -i src/support.c src/swaps.c
+  '';
+
+  postInstall = ''
+    mkdir $out/bin
+    mv $out/sbin/swapspace $out/bin/swapspace
+    rmdir $out/sbin
+    rm -r $out/var
```

Comment:
```suggestion
    rm -r $out/sbin $out/var 
```

Reply:
I'd prefer to keep them separate because sbin should be empty and then it will fail if there's an extra executable